lib/opensusebasetest.pm

```diff
@@ -552,8 +553,11 @@ sub post_fail_hook {
             record_info("no $program", "Could not find '$program' on the system", result => 'fail') && die "$program does not exist on the system";
         }
     }
-    return unless $self->{in_wait_boot};
-    if (wait_serial qr/Reached target Shutdown/) {
+    return unless ($self->{in_boot_desktop} || $self->{in_wait_boot});
+    if ((get_var('BACKEND') // '') eq 'svirt') {
```

Comment:
Why should we limit ourselves to the backend "svirt"?

Reply:
okay, I can remove this limitation, I thought this is only svirt relavent :)